src/DatabaseSchema.php

```diff
@@ -27,5 +27,7 @@ public function __construct($version = 0.1)
         parent::__construct($version);
 
         $this->addModel('User', User::class);
+        $this->addModel('UserPastPassword', UserPastPassword::class);
```

Comment:
I think you use "Previous" a lot in other places as opposed to "Past". I prefer previous as it's clearer - could you refactor this easily?

Reply:
Hi Bert that is no problem I will refactor this.

Reply:
This model have now been changed with the name User Previous Password as I agree the terminology fits better with other descriptions that I have used.